### PR TITLE
bugfix - UCHV-106 - phone number validation

### PR DESCRIPTION
**Pull request by joefitter**
* removed numeric validator from phone number fields
* added custom validator which allows brackes, spaces, plus and hyphen chars

### CODE REVIEW

File: apps/complaints/fields.js [hunk 1]
```diff
@@ -64,7 +68,7 @@ module.exports = {
 	},
 	'phone-number': {
 		mixin: 'input-number',
-		validate: 'numeric'
+		validate: [phoneNumber]
```
Comment (joefitter) [inline apps/complaints/fields.js]: this needs to be an array due to a bug in hof-form-controller

File: apps/complaints/translations/src/en/validation.json [hunk 2]
```diff
@@ -84,7 +84,7 @@
   },
   "phoned-from": {
     "required": "Enter the telephone number that the call was made from",
-    "numeric": "Enter a valid telephone number"
+    "phoneNumber": "Enter a valid telephone number"
```
Comment (lennym) [inline apps/complaints/translations/src/en/validation.json]: Does this work by taking the name of the function? That's clever!

File: apps/complaints/fields.js (hunk 1 above)
Comment (lennym) [inline apps/complaints/fields.js]: Raise an issue?

File: apps/complaints/translations/src/en/validation.json (hunk 2 above)
Comment (joefitter) [inline apps/complaints/translations/src/en/validation.json]: I wrote it all by myself 😊

Comment (joefitter) [inline apps/complaints/translations/src/en/validation.json]: not clever enough to accept a function not in an array though :(

File: apps/complaints/fields.js (hunk 1 above)
Comment (joefitter) [inline apps/complaints/fields.js]: yeah, I was gunna, but got distracted. Ooh look! Something shiny!

Comment (lennym) [inline apps/complaints/fields.js]: I know them feels... now where was I?

File: apps/complaints/translations/src/en/validation.json (hunk 2 above)
Comment (lennym) [inline apps/complaints/translations/src/en/validation.json]: Nice work.

File: apps/complaints/fields.js (hunk 1 above)
Comment (joefitter) [inline apps/complaints/fields.js]: https://github.com/UKHomeOfficeForms/hof-form-controller/issues/7

**Pull request merged**